lib/galaxy/config/__init__.py

```diff
@@ -800,6 +823,64 @@ def __init__(self, **kwargs):
                 'filters': ['stack']
             }
 
+    def _set_reloadable_properties(self, kwargs):
+        for key, default_value in RELOADABLE_CONFIG_OPTIONS.items():
+            method = self._get_property_set_method(key)
+            value = kwargs.get(key) or default_value
+            method(value)
```

Comment:
You could also do `setattr(self, key, value)` and skip the boilerplate below.
The little bit of processing that `_set_admin_users ` does could be replaced by a property + setter.

Reply:
I didn't think of it! It does get rid of a lot of boilerplate. You mean something like this?
```python
def _set_reloadable_properties(self, kwargs):
    for key, default_value in RELOADABLE_CONFIG_OPTIONS.items():
        value = kwargs.get(key) or default_value
        setattr(self, key, value)

def update_reloadable_property(self, key, value):
    if key in RELOADABLE_CONFIG_OPTIONS:  # check again because this is critical
        setattr(self, key, value)

@property
def admin_users(self):
    return self._admin_users

@admin_users.setter
def admin_users(self, value):
    self.admin_users = value
    self.admin_users_list = [u.strip() for u in self.admin_users.split(',') if u]
```

Reply:
Yep, though the setter line should be
`self._admin_users = value`

Reply:
Of course! whoops!
Will push commit in a few minutes. Thanks for the thorough review!